File: SunkiojiDalis/src/Items/Equipables/AbstractEquipable.ts
import { Item } from '../Item';

export abstract class AbstractEquipable extends Item {
  constructor(guid: string, itemData: any) {
    super(guid, itemData);
  }

  public Equip(): number {
    return 0;
  }
  public Unequip(): number {
    return 0;
  }
}

File: SunkiojiDalis/Models/Items/Equipables/AbstractEquipable.cs
namespace SignalRWebPack{
  public abstract class AbstractEquipable: Item {
    public AbstractEquipable(int Id = -1, string Sprite = "", int Weight = -1, int Quantity = -1, int X = -1, int Y = -1, int BelongsTo = -1, string areaId = "") : base(Id, Sprite, Weight, Quantity, X, Y, BelongsTo, areaId){}
    public int Equip(){
      return 0;
    }
    public int Unequip(){
      return 0;
    }
  }
}

